pkgs/development/python-modules/aeidon/default.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  gettext,
+  flake8,
+  isocodes,
+  pytestCheckHook,
+  charset-normalizer,
+}:
+
+buildPythonPackage rec {
+  pname = "aeidon";
+  version = "1.15";
+
+  src = fetchPypi {
+    pname = "aeidon";
+    inherit version;
+    sha256 = "sha256-qGpGraRZFVaW1Jys24qvfPo5WDg7Q/fhvm44JH8ulVw=";
+  };
+
+  nativeBuildInputs = [
+    gettext
+    flake8
+  ];
+
+  dependencies = [ isocodes ];
+
+  installPhase = ''
+    runHook preInstall
+    python setup.py --without-gaupol install --prefix=$out
```

Comment:
Use `setupPyBuildFlags`.

Reply:
I had trouble doing that, as just adding `setupPyBuildFlags = [ "--without-gaupol" ]` results in a working aeidon build, but its reverse dependency `translate-toolkit` fails then due to missing directories. Any ideas?

Reply:
fixed in #338224